lib/pry/commands/clear_screen.rb

```diff
@@ -0,0 +1,14 @@
+class Pry::Command::ClearScreen < Pry::ClassCommand
+  match 'clear-screen'
```

Comment:
Can we make it `clear` by default and add an alias `cls`?

Reply:
it is named `clear-screen` to avoid collisions with local variables/method names.

Reply:
eg,
```
[1] pry(main)> cd ""
[2] pry(""):1> clear
=> ""
[3] pry(""):1>
```

Reply:
While it's a valid example, that's only one method and I have a hunch this collision is unlikely to happen very often. On the contrary, people are familiar with `cls` or `clear` and this is what the majority will try first only to discover this isn't a thing.

Reply:
i do not agree. String is a core class, and therefore "clear" is a core method Ruby name.
"clear" is too common of a potential local or method variable name, same for "cls". 
"clear<TAB>" expands to "clear-screen", too.
It is best to avoid collisions as much as possible.

Reply:
example 2.
```
[1] pry(main)> cd []
[2] pry(#<Array>):1> clear
=> []
```
same is true for Hash, Set, SortedSet. That's five examples.